### ISSUE TITLE
remarque concernant le calcul empreinte kilomètre voiture

### ISSUE THREAD

**Issue by publicodes** [betagouv/ecolab-data#134]
le test ne prend en compte que la consommation du véhicule dans prendre en considération le type du véhicule (petite voiture ou SUV) cela n'est-il pas susceptible d'introduire une distorsion dans le résultat ?

**Comment by laem** [comment]
Oui, cela influence grandement la construction du véhicule, probablement du simple au triple. Une twingo est fortement surestimée, et un SUV Jeep est complètement sous-estimé, pas bien ! 

https://github.com/betagouv/ecolab-data/blob/master/data/transport.yaml#L91

**Comment by lbranaa** [comment]
@laem, cela est pris en compte maintenant avec cette question, non ?
![Capture d’écran 2023-01-12 à 16 26 40](https://user-images.githubusercontent.com/120486950/212108627-0ed3b32f-1690-405e-a092-50ffba12ada3.png)

**Comment by laem** [comment]
Oui, je crois que ça répond en partie à la question. C'est peu précis, mais on a quand même deux empreintes de construction différentes

https://github.com/datagir/nosgestesclimat/blob/master/data/transport/transport%20.%20voiture.yaml#L319

On pourrait faire mieux, ce serait même bien je pense, vu l'enjeu.
